### PR TITLE
support for projection_x/y_angular_coordinate added

### PR DESCRIPTION
**Pull request by lllliso**
The functionality added is the support of geostationary projection for CF => 1.9 the support for geostationary following CF<1.9 is preserved. Also is preserved the support for gestationary not totally CF. Test passed :). If the code is too inflate no worries just reject the pull request.

### CODE REVIEW

**Comment by maartenplieger** [comment]
Hi @lllliso do you have a testfile which works with the new code? I would like to try to add a functional test with this feature.

**Comment by lllliso** [comment]
Hi @maartenplieger, I sent to you the file via weTransfer. For this particular file when updating the dB the following message is expected to be included in the logs:
[D:039:pid13294: adagucserverEC/CDataReader.cpp:594 CDataReader]                Standard Name of the nx variable: projection_x_angular_coordinate
[D:040:pid13294: adagucserverEC/CDataReader.cpp:596 CDataReader]                -----------------
[D:041:pid13294: adagucserverEC/CDataReader.cpp:635 CDataReader]                Assuming  CF 1.9. geostationary projection, keeping radians as units
[D:042:pid13294: adagucserverEC/CDataReader.cpp:638 CDataReader]                Overwriting the projection string with: +proj=geos +lon_0=0.000000 +lat_0=0.000000 +h=1.000000 +a=0.178231 +b=0.177633 +sweep=y

**Comment by lllliso** [comment]
I hope if helps for the test :)

**Comment by lllliso** [comment]
If you need help with the test, just let me know. I will be travelling to Darmstadt from 9 to 11 and I think that I will have a lot of time in the airport :)

**Comment by lllliso** [comment]
Hi Maarten I pushed the test but I left it commented. There are something wrong with the image or the image compression on my system. I tried to regenerate your images for testdata.nc and the problem is still present (your test fails with the image that I produce for testdata.nc) i thing that is something about the png compression, but not sure...
Please can you produce the image for the test?

**Comment by maartenplieger** [comment]
Hi @lllliso looking good! I will check now why the test sometimes fails.

**Comment by maartenplieger** [comment]
> 
here it is:
test_WMSGetMap_testgeosnc

You can do this yourself by first running the tests and let it fail. Then do
```
cp tests/testresults/TestWMS/test_WMSGetMap_testgeosnc.png tests/expectedoutputs/TestWMS/
```
And then run the tests again.

If you add this and uncomment the tests, I think it is OK!

File: Adaguc installation ubuntu18 dev mode.txt [hunk 1]
```diff
@@ -0,0 +1,167 @@
+/// instructions for new instalation on ubuntu 18
```
Comment (maartenplieger) [inline Adaguc installation ubuntu18 dev mode.txt]: This file should not belong here.

Comment (lllliso) [reply Adaguc installation ubuntu18 dev mode.txt]: Yes sure I'll move it outside

**Comment by lllliso** [comment]
Hi Maarten, now the test is working fine, thank you for the guidance :)

**Review by maartenplieger — APPROVED** [review]
Looks good. Tested with files from https://noaa-goes16.s3.amazonaws.com/index.html#ABI-L2-CMIPF/2022/346/09/ as well.

**Pull request merged**